introduction.md

```diff
@@ -0,0 +1 @@
+Hello dojo! My name is Maciej and I'm an aspiring coder from Warsaw, Poland.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.